javascript/javascript1/week1/Ez-Namey.js

```diff
@@ -0,0 +1,7 @@
+const firstWords = ['Mercury', 'Venus', 'Earth', 'Mars', 'Jupiter', 'Saturn', 'Uranus', 'Neptune', 'Stars', 'Sun'];
+const secondWords = ['Dog', 'Puppy','Turtle', 'Rabbit', 'Parrot', 'Cat', 'Kitten', 'Goldfish', 'Mouse', 'Hamster'];
+const FirstRandomNumber = Math.floor(Math.random() * 10) + 0;
+const SecondRandomNumber = Math.floor(Math.random() * 10) + 0;
+let startupName;
+startupName = firstWords[FirstRandomNumber] + secondWords[SecondRandomNumber];
```

Comment:
These lines should be combined:
`const startupName = firstWords[firstRandomNumber] + secondWords[secondRandomNumber];`

Reply:
You should use a `const` since this value is never reassigned.